docs/rules/index.md

```diff
@@ -0,0 +1,40 @@
+# Available Rules
+
+- :star: mark: the rule which is enabled by `plugin:@intlify/vue-i18n/recommended` preset.
+- :black_nib: mark: the rule which is fixable by `eslint --fix` command.
+
+## Recommended
+
+<!--prettier-ignore-->
+| Rule ID | Description |    |
+|:--------|:------------|:---|
+| [@intlify/vue-i18n/<wbr>no-deprecated-i18n-component](./rules/no-deprecated-i18n-component.html) | disallow using deprecated `<i18n>` components (in Vue I18n 9.0.0+) | :black_nib: |
```

Comment:
I think we probably need to change the path as well.

```suggestion
| [@intlify/vue-i18n/<wbr>no-deprecated-i18n-component](./no-deprecated-i18n-component.html) | disallow using deprecated `<i18n>` components (in Vue I18n 9.0.0+) | :black_nib: |
```

Reply:
done :)